app/design/adminhtml/default/default/template/boltpay/update_popup.phtml

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2016-2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/* @var Bolt_Boltpay_Block_Adminhtml_UpdatePopup $this */
+$downloaderUrl = $this->getUpdater()->getConnectManagerURL();
+$updateSeverity = $this->getUpdater()->getUpdateSeverity();
+$updateVersion = $this->getUpdater()->getLatestVersion();
+$patchVersion = $this->getUpdater()->getPatchVersion(Bolt_Boltpay_Helper_ConfigTrait::getBoltPluginVersion());
+$isPatch = $updateSeverity == 'patch';
+?>
+<script type="text/template" id="bolt_update_popup_template">
+    <h3>
+        <?php echo $this->__(
+            'Before upgrading Bolt plugin on production, this process and code should be tested in a staging environment.'
+        ); ?>
+    </h3>
+    <h3>
+        <?php echo $this->__(
+            'You can find our guide to production readiness testing at <a href="%1$s">%1$s</a>.',
+            'https://support.bolt.com/hc/en-us/articles/360046487254-Production-Readiness'
```

Comment:
Are we sure that this link is stable enough to include it source code?

Reply:
Let's hope that we stabilize the link for this as is serves no good purpose to continue to change this over and over again.  I will bring this up as an issue in MX Weekly.

@vitaliyreznikov Do you have a proposal of how to dynamically determine this link?  I, currently do not.

As is, assuming that we do the 3-week release schedule and merchants update, this will only be out-of-date for a short time.  Otherwise, we can hope that if we decide to change the link for whatever reason, we also set up a 301 permanent redirect so that there is no issue with relying on the stability of official Bolt support links.

Reply:
I asked Eric about this link.